src/components/NewsArticle/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {
+    Text,
+    VStack,
+    Link
+} from '@chakra-ui/react';
+
+const NewsArticle = ({date, title, article}) => {
```

Comment:
I think the news article component is small enough to be added to the top of the NewsCard component